src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void draw() {
+    }
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `draw()` method doesn't seem to be related to the area calculation and thus, it might be better to separate it into a different interface or abstract class.